subsys/bluetooth/controller/ll_sw/pdu.h

```diff
@@ -710,6 +710,26 @@ struct pdu_data_llctrl_conn_param_rsp {
 	uint16_t offset5;
 } __packed;
 
+/*
+ * According to Spec Core v5.3, section 2.4.2.17
+ * LL_CONNECTION_PARAM_RSP and LL_CONNECTION_PARAM_REQ are identical
+ * This is utilized in pdu encode/decode, and for this is needed a common struct
+ */
+struct pdu_data_llctrl_conn_param_req_rsp_common {
```

Comment:
Answer: Yes I think it would, however Thomas and I agreed that could be a bit hard to get that approved in this refactoring, so I went for this less drastic approach;-)